src/entity/sources/SourceCategory.ts

```diff
@@ -22,4 +23,16 @@ export class SourceCategory {
 
   @UpdateDateColumn()
   updatedAt: Date;
+
+  @Column({
+    type: 'text',
+    update: false,
+    insert: false,
+    nullable: false,
+    unique: true,
+    generatedType: 'STORED',
+    asExpression: `trim(BOTH '-' FROM regexp_replace(lower(trim(COALESCE(LEFT(source_category.title,100),''))), '[^a-z0-9-]+', '-', 'gi'))`,
```

Comment:
Do we need the COALESCE and LEFT still? 
Think just lower trim should do right?

Reply:
LEFT needed to still keep max 100 chars in a slug I think

Reply:
Cool. Wouldn't really change anything without COALESCE. Actually better to have it than not.